### ISSUE TITLE
Some questions on FedDF results 

### ISSUE THREAD

**Issue by honggudrnjs** [epfml/federated-learning-public-code#11]
Hello, Lin and Kong, the first authors of the paper, FedDF.
In advance, thanks for sharing your amazing work, FedDF.
I wanted to apply your amazing work FedDF in my project.
However, some results in the paper were confusing.

In Table 1 of the main paper, when C = 0.4, alpha=0.1, and local epochs were set to 40, 75% accuracy should be achieved at 39 +- 10 communication round.

However, the table 2 says that 71.36 +- 1.07 % accuracy is achieved at 100 communication round with the same condition.
Since 71.36 is much smaller than 75, the results confused me a lot. Could you explain why this happens?

**Comment by tlin-taolin** [comment]
Hey, thank you for your interest. 

We cannot recall all details after 3 years, and we believe hyper-parameters and variance cause the difference.

Indeed FedDF is mainly very good at the initial training phase, and will soon become saturated (see other figures in the paper)
